src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -334,8 +338,9 @@ const useGroupInfo = () => {
   const handleMessageAnonymously = async () => {
     try {
       setIsLoadingInitChat(true);
-      await handleAnonymousMessage(selectedUser);
+      await handleAnonymousMessage(selectedUser, channelId, 'chat');
     } catch (e) {
+      console.log(e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the `handleMessageAnonymously` function, you've added error logging with `console.log(e)`. While this is fine for development, it's not a good practice for production code as it can expose sensitive information. Consider using a dedicated error reporting service that respects user privacy.

Also, you've added a new argument `channelId` to the `handleAnonymousMessage` function call. Ensure that the function definition is updated accordingly and handles this new argument properly.
```diff
- await handleAnonymousMessage(selectedUser);
+ await handleAnonymousMessage(selectedUser, channelId, 'chat');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->